### ISSUE TITLE
This webpage is not available

### ISSUE THREAD

**Issue by creativetec** [snipe/snipe-it#1021]
Hi

After following all the instruction to install onto IIS. 
http://localhost
It come out a page " This webpage is not available"

i follow everything stated in the doc.

![image](https://cloud.githubusercontent.com/assets/13635883/9085593/b62def54-3bb0-11e5-9cd6-57c1eecd7610.png)

**Comment by madd15** [comment]
did you set it up to use `https`? does anything load when you goto `http://localhost`?

**Comment by snipe** [comment]
Any update?

**Comment by tsilman-marquette** [comment]
I have same problem. I am new to server site management and have probably set something wrong. I have followed the directions twice checking each step. 
From what I can see, I request a http connection and it redirects to a https connection with the above resulting.

**Comment by tsilman-marquette** [comment]
Hmmm
I looked at the URL rewrite and it redirects to https by rule. Is this the problem?

**Comment by tsilman-marquette** [comment]
I'm dumb. I think I have this running in production mode instead of development mode.

**Comment by snipe** [comment]
If you're not planning on writing any code for it, production mode is the way you want to go. The rewrite https rules are commented out, unless you un commented them by accident.

**Comment by snipe** [comment]
@tsilman-marquette in your `app/config/production/app.php`, does your url start with http or https? 

The only two ways it's going to redirect to https (outside of some weird server config) is if you un-commented the rewrite rules, or if your `url` value is https in `app/config/production/app.php`.

**Comment by tsilman-marquette** [comment]
app.php = > copied from        'url' => 'http://localhost/', 
Hmmm. I'm going to vote weird server config... I probably did something.

The URL rewrite rules were imported from .htaccess as per the documentation for Windows install. 
Copy/Paste follows
<IfModule mod_rewrite.c>
    <IfModule mod_negotiation.c>
        Options -MultiViews
    </IfModule>

    RewriteEngine On

    # Uncomment these two lines to force SSL redirect
    # RewriteCond %{HTTPS} off
	# RewriteRule (.*) https://%{HTTP_HOST}%{REQUEST_URI} [R=301,L]

    # Redirect Trailing Slashes...
    RewriteRule ^(.*)/$ /$1 [L,R=301]

    # Handle Front Controller...
    RewriteCond %{REQUEST_FILENAME} !-d
    RewriteCond %{REQUEST_FILENAME} !-f
    RewriteRule ^ index.php [L]
</IfModule>

I'm running Windows Server 2012 and IIS 8.0.92 if it makes any difference. I've tried turning off the rules to see if the behavior would change and nothing occurred.

At this point, I am assuming that I missed something during the install.

**Comment by tsilman-marquette** [comment]
Huh. Found the problem. It was me.
From what I can tell, the .htaccess import created the proper rules for secure operation. I disabled the rule and tested. Nothing changed... Course, it would have changed if I had remembered to clear the cache of the browser I was using for testing...  Doh. 
Everything is now working and is ready for testing. I plan to put it into production later this week. 
Thank you for your patience.
